llvm/test/CodeGen/X86/scmp.ll

```diff
@@ -0,0 +1,985 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc < %s -mtriple=x86_64-unknown-unknown | FileCheck %s
```

Comment:
Please can you add i686-- test coverage as well

Reply:
Done